packages/governance/src/typeGuards.js

```diff
@@ -0,0 +1,285 @@
+// @ts-check
+
+import { M } from '@agoric/store';
+import { TimestampShape } from '@agoric/swingset-vat/src/vats/timer/typeGuards.js';
+
+export const ChoiceMethodShape = M.or('unranked', 'order');
+export const QuorumRuleShape = M.or('majority', 'no_quorum', 'all');
+export const ElectionTypeShape = M.or(
+  'param_change',
+  'election',
+  'survey',
+  'api_invocation',
+  'offer_filter',
+);
+
+const makeHandleShape = name => M.remotable(`${name}Handle`);
+
+export const TimerShape = makeHandleShape('timer');
+export const InstanceShape = makeHandleShape('instance');
```

Comment:
Btw, zoe/src/typeGuards.js exports this as `InstanceHandleShape`. However, zoe as a whole doesn't export it. I'm thinking it should. When I did this I missed the `timer: M.remotable(),` on like 33 of that file, that should have been replaced with the `TimerShape` you define above.

I think a package like Zoe should export the shapes that correspond to concepts defined by that package. But that refactor is distinct from this PR. No change suggested.